rigex-front-end/src/components/vital-resources/AboutUs/AboutUs.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import faker from 'faker'
+import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
+
+import './aboutUsStyle.scss'
+
+const AboutUs = () => {
+
+  const randomImage = faker.image.image()
+
+  return (
+    <div className="about-section">
+      <div className="image-container">
+        <img src={randomImage} />
+      </div>
+      <div className="text-container">
+        <span className="title">About Us</span>
+        <span className="name">OILDROP GLOBAL Technology Suite</span>
+        <span className="paragraph">
+          Using a latest technology leader in a gloabl market in a prescion testing, wetsock managment and fuel restoration technologies. Our expertise enables customers to achieve the tighest storage systems, lowest fuel losses and most accurate wetstock management. Customers also beenfit from optimal pump speed, more precise re-strapping of guages, the highest compliance and cleanest fuel.
```

Comment:
A few words misspelled (benefit, management). Just double check that real quick.

Also, this text is in a span with a classname called paragraph. why not just rap in it in a <p> tag?

Reply:
This whole section can be updated now with data from the backend, this was just to get something up that looked like the design specification